### DISCUSSION TITLE
Future of VisualFSM

### DISCUSSION THREAD

**Discussion by imberny** [imberny/VisualFSM#2] [General]
Is anybody using this? If so what are you using it for? And what improvements would you like to see?

**Comment by higgs-bosoff** [comment]
I just develop games for fun so I was going around seeing the various plugins for Godot that do this, and this seemed to me like the best one in terms of being simple, flexible (different trigger types) and having a good GUI. A lot of the ones on the Asset Library are older than the GraphEdit component, so the GUI feels fiddly by comparison. 

Personally my interest was in trying to use it for a piloting AI in a space shooter, but nothing done yet!

**Reply by imberny** [reply]
Nice! Well let me know if you encounter limitations with this plugin, and I'll see if I can do something about it. It's more fun when you know someone will end up using it.

**Reply by imberny** [reply]
Btw the update is currently pending approval in the asset lib. But you can also just get the files directly from here.

**Comment by BigDC** [comment]
I am using it, I am using it for a Platformer game I am making! 
How can I call/change a state from a SCRIPT trigger?
